tests/test_selector.py

```diff
@@ -49,3 +51,33 @@ def test_filtered_query(self):
     def test_filtered_query_no_match(self):
         outer_id_list = self.selector.ids_of_outer_divs_with_inner_text('This does not match anything')
         self.assertEquals(outer_id_list, [])
+
+
+class ScrollTest(WebAppTest):
+    """
+    Test scrolling to element
+    """
+
+    def setUp(self):
+        super(ScrollTest, self).setUp()
+
+        self.long_page = LongPage(self.browser)
+        self.long_page.visit()
+
+    def test_scroll(self):
+        self.assertEquals(self._get_window_position(), 0)
+        self.long_page.scroll_to_element('#element_after_long_part')
+        # Different browsers, CI systems, and resolutions may present the
+        # element in varying locations. Use a greater-than instead of an equals.
```

Comment:
One other suggestion would be to get the actual position of the element for the comparison. You can do this with jQuery using `offset`:

```
$('#element_after_long_part').offset().top
```